src/Compatibility/ControlGallery/src/Issues.Shared/Bugzilla31114.cs

```diff
@@ -325,9 +325,9 @@ void Init(bool fast)
 			}
 
 
-			protected override void OnPropertyChanged(string propertyName = null)
```

Comment:
Can we make anything that's modifying a public API internal or `private protected` for now? Then we can look at making something public for NET7?

Reply:
The problem is that we can't have two `OnPropertyChanged` methods as I originally wanted. Should we remove `OnPropertyChanged` from the public API then?